fluentgen/src/main/java/com/azure/autorest/fluent/mapper/ExampleParser.java

```diff
@@ -228,7 +228,7 @@ private static void addMethodParametersToMethodExample(List<MethodParameter> met
             ExampleNode node = parseNodeFromParameter(proxyMethodExample, methodParameter);
 
             if (node.getObjectValue() == null) {
-                if (methodParameter.getClientMethodParameter().getIsRequired()) {
+                if (methodParameter.getClientMethodParameter().isRequired()) {
```

Comment:
nit, I think rename is great. Though personally prefer doing it in a separate PR and get merged very soon.

Again, I am fine with it, but you might get to fix compile error when refresh main, in the long PR.